src/viewer/Entity.tsx

```diff
@@ -142,14 +143,20 @@ class Entity extends React.Component<Props & CardsProps & PortalProps, State> {
 						x,
 						y,
 					});
+
+					gaEventTimeout = setTimeout(() => {
```

Comment:
You don't need to duplicate this here. Let's just reuse `this.state.showStatistics` - we could rename it to `isMeaningfulHover` to be more clear. It's also used to fire off an HTTP request to HSReplay.net after the timeout that's set further up in this file. Then you would just do the following in `componentDidUpdate`:

```typescript
if (this.state.isHovering && !prevState.isMeaningfulHover && this.state.isMeaningfulHover) {
  ga("send", "event", "Hover", this.props.dbfId);
}
```

Reply:
hum, yeah, reuse that statistics timeout is much better!